src/scripts/OSFramework/OSUI/Pattern/AccordionItem/scss/_accordion-item.scss

```diff
@@ -179,6 +179,11 @@
 			&-collapsed {
 				height: 0;
 				visibility: hidden;
+
+				// This is to avoid reflowing the content when collapsed
+				& > div {
+					display: none;
+				}
```

Comment:
Maybe we could do this only on Chrome, with the .chrome class.

Reply:
We could yes, but why not having this for all browsers? Do you see a problem with it?

Reply:
If the customers have some kind of opacity or transform animaiton for the accordionItem content, when its open, display: none will disable those

Reply:
But I see you say about the transitionEnd, probably it guarantees this would still work as expected yes 👍